node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -982,8 +982,42 @@ var _updateOrCreateOrgUnit = function(appId, orgUnitToImport, parent, callback)
             'published': orgUnitToImport.published,
             'parentId': parent.id
         };
-        return createOrgUnit(appId, orgUnitToImport.displayName, orgUnitToImport.type, parent.GroupId, opts, callback);
+
+        // TODO: Refactor this check out of the DAO
+        _getOrCreateGroup(appId, orgUnitToImport.type, parent, function(err, groupId) {
```

Comment:
No, if it's a module, we want it to use the part's group. If it's a part, we want a new group created for it. part > module.